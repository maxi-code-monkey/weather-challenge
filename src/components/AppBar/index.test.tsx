import React from "react";
import { render } from "@testing-library/react";
import CustomAppBar, { APP_BAR_TEST_ID } from "./index";

describe("<AppBar />", () => {

  it("Renders correctly", () => {
    const { getByText, getByTestId } = render(<CustomAppBar />);
    const component = getByTestId(APP_BAR_TEST_ID);
    const typography = getByText(/Flow Weather Challenge/);
    
    expect(typography).toBeInTheDocument();
    expect(component).toMatchSnapshot();
  });
});