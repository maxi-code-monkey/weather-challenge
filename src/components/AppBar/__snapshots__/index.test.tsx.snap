// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppBar /> Renders correctly 1`] = `
<div
  class="MuiBox-root css-i9gxme"
  data-testid="app-bar-test-id"
>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
  >
    <div
      class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
    >
      <button
        aria-label="menu"
        class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-edgeStart MuiIconButton-sizeLarge css-ancrnh-MuiButtonBase-root-MuiIconButton-root"
        tabindex="0"
        type="button"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="MenuIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
          />
        </svg>
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </button>
      <div
        class="MuiTypography-root MuiTypography-h6 css-1juivf6-MuiTypography-root"
      >
        Flow Weather Challenge
      </div>
    </div>
  </header>
</div>
`;
